python/cli.rst

```diff
@@ -0,0 +1,68 @@
+##############################
+Python Command Line Interfaces
+##############################
+
+Command line interfaces (CLI) in Python may be implemented using the ``argparse`` module or using `Click`_.
```

Comment:
Why not link ``argparse`` to https://docs.python.org/3/library/argparse.html?

Reply:
Can you add a comment here as to why one choose Click over argparse, or vice versa? It'd be great if the developer reading this page could receive clear guidance about what they ought to do, rather than just a list of the possibilities.